build.sbt

```diff
@@ -227,12 +229,14 @@ libraryDependencies ++= List(
 )
 
 //for aether and compensating for 2.11 modularization
-libraryDependencies <++= scalaBinaryVersion {
-  case "2.10" => Nil
-  case "2.11" => List(ningAsyncHttpClient,
-    "org.scala-lang.modules" %% "scala-xml" % "1.0.4",
-    "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.4"
-  )
+libraryDependencies ++= {
```

Comment:
I see in some places you added extra braces (`{ .... }`) around an expression and in some using the value directly (smaller diff), e.g. https://github.com/spark-notebook/spark-notebook/pull/915/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR304

do you know if these behave some or not?

Reply:
This relates to usage of `.value` notation. It is not allowed (see sbt error few comments below) to use the notation outside the scope of setting expression. It looks like `.value` is not considered to be within setting expression unless it is enclosed by braces or "hidden" inside string substitution. In such cases I need to use additional braces to make sbt happy. I am not sure if this is a sbt bug or not...

Reply:
I don't think additional braces should change a way how it behaves. I am not dead sure however. I wanted to try to see libraryDependencies content for two versions of scala but as scala 2.10 is not supported anymore (exception raised) I didn't...

Reply:
so it seems both is valid now as long as you use `.value`
http://www.scala-sbt.org/0.13/docs/Migrating-from-sbt-012x.html#Migrating+from+the+tuple+enrichments